api/javatests/org/openyolo/api/internal/KnownProvidersTest.java

```diff
@@ -39,6 +39,7 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
+import org.openyolo.api.internal.KnownProviders;
```

Comment:
I don't think this import is necessary, is it?

Reply:
Fixed.